modules/ROOT/pages/flowref-about.adoc

```diff
@@ -3,30 +3,82 @@ ifndef::env-site,env-github[]
+The next diagram shows the order of processing when one flow references another. When the main flow is triggered, the Mule event travels through and executes the flow until the event reaches the Flow Reference. The event then travels through and executes the referenced flow from beginning to end, and then returns to the main flow.
 
-In this example, the Flow Ref component points to a sub-flow called ProcessFiles.
+image::flowref-about-8b5d1.png[]
 
-image::component-flowref-example.png[Flow Ref Configuration Example]
+== Configure a Flow Reference
 
-The diagram below shows the order of processing when one flow references another. When the main flow is triggered, the Mule event travels through and executes the flow until the event reaches Flow Ref. The event then travels through and executes the referenced flow from beginning to end, and returns to the main flow.
+To configure a Flow Reference component, follow these steps in Anypoint Studio:
 
-image::flowref-about-8b5d1.png[]
+. Drag the Flow Reference component from *Mule Palette* to the point in your flow where you want to create the call to the referenced flow:
++
+image:mruntime-flow-ref-create.png[flow-ref-create]
+. Open the Flow Reference properties and specify the flow you want to reference in *Flow name*:
++
+image:mruntime-flow-ref-config.png[flow-ref-config]
+
+== Enrich Content with a Flow Reference Target Variable
+
+You can store the result of a Flow Reference call in a target variable by configuring the following fields in your Flow Reference properties:
+
+* *Target*: Name of the variable that the Flow Reference creates to store the result of the processed payload.
+* *Target Value*: An expression that evaluates against the operation's output, the result of this evaluation is stored in the defined target variable.
+
+=== Considerations About Target Variable
+
+Setting the `target` variable in a Flow Reference component causes the message to remain unchanged. This means that any modification to the payload or to the variables that occur in the referenced flow will revert once the referenced flow finishes executing its processors, returning the payload and variables to its original values.
+
+On the other hand, if you do not define a `target` variable, any modification to the payload or to the variables that occurs in the referenced flow will persist after the referenced flow finishes its execution, changing the actual value of the payload and variables.
+
+=== XML Example
+
+The following XML code represents the project configured in the previous steps, and also defines target variable `flowReferenceVar` which stores the result of the secondary flow called by the Flow Reference component. This variable is then used from the main flow by logging `vars.flowReferenceVar`:
```

Comment:
Because "the previous steps" appear to be rather removed from this text, can we reword by internal xref?
```suggestion
The following XML code represents the project configured in << title >>, adding the target variable `flowReferenceVar,` which stores the result of the secondary flow called by the Flow Reference component. This variable is then used in the main flow by logging `vars.flowReferenceVar`:
```